sharktank/sharktank/types/tensors.py

```diff
@@ -240,6 +239,14 @@ def _clone_with_globals(
             f"InferenceTensor {type(self)} does not implement _clone_with_globals"
         )
 
+    @property
+    def T(self) -> "InferenceTensor":
+        from ..ops import permute
+
+        return permute(
+            self, dims=torch.flip(torch.arange(len(self.shape)), dims=[0]).tolist()
```

Comment:
@stellaraccident, I am not sure how `torch.flip(torch.arange(len(self.shape)), dims=[0]).tolist()` would be handled by model tracing. `torch.arange` returns a `torch.Tensor`. At the end we conver to a `list`. This must not end up as part of an exported model.

Reply:
I fixed this by using more pythonic constructs to generate the reverse dimensions list. This seems to make the torch export tracer happy.